go.mod

```diff
@@ -3,13 +3,14 @@ module github.com/signalfx/ingest-protocols
 go 1.14
 
 require (
-	github.com/apache/thrift v0.13.0
+	github.com/apache/thrift v0.14.1
+	github.com/axw/gocov v1.0.0 // indirect
 	github.com/gobwas/glob v0.2.3
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/protobuf v1.4.3
 	github.com/golang/snappy v0.0.1
 	github.com/gorilla/mux v1.7.4
-	github.com/jaegertracing/jaeger v1.22.0
+	github.com/jaegertracing/jaeger v1.22.1-0.20210317135401-40e96f966d7c
```

Comment:
is there any proper release for jaeger that can be used ?

Reply:
Nope, we have to use master or else it is impossible to make work since the thrift version we need (0.14.1) is completely incompatible with the latest Jaeger release.